core/services/pipeline/runner_test.go

```diff
@@ -599,9 +599,8 @@ ds1 -> ds1_parse;
 		var se []models.JobSpecErrorV2
 		err = db.Find(&se).Error
 		require.NoError(t, err)
-		require.Len(t, se, 2)
+		require.Len(t, se, 1)
```

Comment:
Huh, why did this change?

Reply:
Tests fail without it

Reply:
Because we now no longer falsely log context shutdowns as timeout errors under certain circumstances

Reply:
Gotcha, awesome